content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/4.0/release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/4.0/release.png
+changelogItems:
+---
+
+WIP INTRO: 
+
+We've spent 9 years building the underlying foundation of code search to be fast, secure, and scalable across ~10^11 lines of code for the world's most demanding software teams. Now, we’re hard at work making Sourcegraph even smarter.
+
+Sourcegraph 4.0 includes over a dozen updates and new features, and is the first step toward Sourcegraph becoming a [code intelligence platform](blog/code-search-to-code-intelligence).
+
+Read on to discover everything that shipped in Sourcegraph 4.0.
```

Comment:
```suggestion
Read on to discover everything in Sourcegraph 4.0.
```

and recommend adding a table of contents for easy scannability, and then the CTA (which is Cloud signup and Cloud tease) right after that, then all the sections

Reply:
Working on the table of contents and CTA too.